server/src/main/java/org/tbbtalent/server/model/es/CandidateEs.java

```diff
@@ -156,7 +155,6 @@ public class CandidateEs {
 
     private String partner;
 
-    @Field(type = FieldType.Text)
```

Comment:
Wondering why this annotation is no longer needed. It is used elsewhere.

Reply:
This forces the index mapping for full text search only. Removing it means elastic search includes both full text and keyword mappings by default. The keyword mapping allows term queries for a given field (as well as full text searches).